src/components/tabs/Tabs.test.tsx

```diff
@@ -0,0 +1,78 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { axe } from 'jest-axe'
+import * as React from 'react'
+import { IdProvider } from '@radix-ui/react-id'
+
+import { Tabs } from '.'
+
+const TabsTest = ({ defaultValue = 'tab1' }) => (
+  <IdProvider>
+    <Tabs defaultValue={defaultValue}>
+      <Tabs.TriggerList>
+        <Tabs.Trigger value="tab1">Trigger 1</Tabs.Trigger>
+        <Tabs.Trigger value="tab2">Trigger 2</Tabs.Trigger>
+      </Tabs.TriggerList>
+      <Tabs.Content value="tab1">Important content for tab 1</Tabs.Content>
+      <Tabs.Content value="tab2">Important content for tab 2</Tabs.Content>
+    </Tabs>
+  </IdProvider>
+)
+
+describe('Tabs component', () => {
+  it('renders', async () => {
+    const { container } = render(<TabsTest />)
+
+    expect(container).toMatchSnapshot()
+  })
+
+  it('has no programmatically detectable a11y issues', async () => {
+    const { container } = render(<TabsTest />)
+
+    expect(await axe(container)).toHaveNoViolations()
+  })
+
+  it('it makes the correct tab content visible, based on the `defaultValue` prop', async () => {
+    render(<TabsTest defaultValue="tab2" />)
+    const tab1content = await screen.findByText('Important content for tab 1')
+    const tab2content = await screen.findByText('Important content for tab 2')
```

Comment:
```suggestion
    const tab2content = await screen.findByText('Important content for tab 2')
    
```

Just adding a bit of space for readability. Very good test by the way

Reply:
I can't tell what the change here is :))

Reply:
just adding a bit of space between the lines to make them a bit more legible

Reply:
I adjusted the spacing on the entire file.